packages/figma/src/codegen/targets/react/import.ts

```
@@ -0,0 +1,45 @@
+  "IdentityPlaceholder",
+  "LinkContent",
+  "NotificationBadge",
+  "Skeleton",
+  "HStack",
+  "VStack",
+  "Text",
+  "VisuallyHidden",
+]);
+
+export function createImportSyntaxService() {
+  function makeImportSyntax(
```

Biome (1.9.4)

[error] This function is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.

(lint/correctness/noUnusedVariables)